.travis.yml

```diff
@@ -1,16 +1,25 @@
 ---
+dist: trusty
```

Comment:
The [build environment page](https://docs.travis-ci.com/user/ci-environment/) is a little hazy on this, but iirc specifying distribution is reserved for virtual machine builds. I probably need to clarify this for myself, but it's a good conversation to work through.